src/utils/wallets.ts

```diff
@@ -48,3 +51,10 @@ export const isWalletUnlocked = async (walletName: string): Promise<boolean> =>
 
   return false
 }
+
+export const getEthHashInfoPrefix = (
+  shortName?: ChainInfo['shortName'],
+  chainId?: ConnectedWallet['chainId'],
+): string | undefined => {
+  return shortName || (chainId ? Object.entries(chains).find((chain) => chain[1] === chainId)?.[0] : undefined)
+}
```

Comment:
I wouldn't place it here, as it's only used in that one place.
This is short enough for an inline call IMHO.
```
shortName || Object.entries(chains).find(([_, val]) => val === chainId)?.[0]
```

Reply:
It's used in two places though.

Reply:
I would suggest extracting only the find part, as it's used in many more places in the app.
I.e.
```
const getChainPrefix = (chainId) => Object.keys(chains).find(key => chains[key] === chainId
```
And it shouldn't be in `wallets.ts` obviosuly.

Reply:
Just saw you did exactly that in #1619. :)